imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -78,6 +84,11 @@ def process_and_write_data(
     if not packet_data:
         return []
 
+    # TODO: Rework attrs to be better
+    raw_attrs["Data_version"] = data_version
```

Comment:
This is a temporary fix, I wanted to run L1A in the cloud today for MAG but I have rewritten the way these attributes work in my WIP branch.